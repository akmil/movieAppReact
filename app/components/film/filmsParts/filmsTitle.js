import React from 'react';

export const FilmsTitle = ({title, rate}) => (
    <h1 className='films-title mb-0'>
        {title}
        <span className={'badge badge-transparent ml-2'}>{rate}</span>
    </h1>
);