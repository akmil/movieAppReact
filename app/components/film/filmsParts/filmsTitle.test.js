import React from 'react';
import { shallow } from 'enzyme';
import {FilmsTitle} from './filmsTitle';
import renderer from "react-test-renderer";
// import {SearchButton} from './searchButton';
// import renderer from 'react-test-renderer';

describe('<films FilmsTitle/>', () => {
    it('should render <FilmsTitle /> components', () => {
        // const wrapper = shallow(<FilmsSortBy />);
        const wrapper = renderer.create(<FilmsTitle/>);
        expect(wrapper).toMatchSnapshot();
    });
});