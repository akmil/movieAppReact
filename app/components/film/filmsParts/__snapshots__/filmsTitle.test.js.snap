// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<films FilmsTitle/> should render <FilmsTitle /> components 1`] = `
<h1
  className="films-title mb-0"
>
  <span
    className="badge badge-transparent ml-2"
  />
</h1>
`;
